src/db/migrations/0002_fix_report_json_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE lighthouse_audits ALTER COLUMN report_json TYPE json
```

Comment:
PG's jsonb type does not preserve the order of keys.
The report generator relies on the "categories" property to be in a specific order.